python/cugraph/cugraph/gnn/dgl_extensions/cugraph_store.py

```diff
@@ -46,10 +34,13 @@ class CuGraphStore:
     """
 
     def __init__(self, graph, backend_lib="torch"):
-        if isinstance(graph, (PropertyGraph, MGPropertyGraph)):
+
+        if type(graph).__name__ in ["PropertyGraph", "MGPropertyGraph"]:
             self.__G = graph
         else:
             raise ValueError("graph must be a PropertyGraph or MGPropertyGraph")
+
+        BaseCuGraphStore.__init__(self, graph)
```

Comment:
I think this is usually preferred:
```suggestion
        super().__init__(graph)
```